gradle/libs.versions.toml

```diff
@@ -16,13 +17,14 @@ compileSdk = "34"
 androidx-annotations = { module = "androidx.annotation:annotation", version = "1.8.2" }
 androidx-appcompat = { module = "androidx.appcompat:appcompat", version = "1.7.0" }
 androidx-activity = { module = "androidx.activity:activity", version = "1.9.1" }
-androidx-compose-ui-android = { module = "androidx.compose.ui:ui-android", version = "1.6.8" }
+androidx-compose-ui-android = { module = "androidx.compose.ui:ui-android", version.ref = "compose" }
 androidx-recyclerview = { module = "androidx.recyclerview:recyclerview", version = "1.3.2" }
 
 bytebuddy-agent = { module = "net.bytebuddy:byte-buddy-agent", version.ref = "bytebuddy" }
 bytebuddy-core = { module = "net.bytebuddy:byte-buddy", version.ref = "bytebuddy" }
 
-composeUi-material = { module = "androidx.compose.material:material", version = "1.6.8" }
+compose-runtime = { module = "androidx.compose.runtime:runtime", version.ref = "compose" }
+composeUi-material = { module = "androidx.compose.material:material", version.ref = "compose" }
 composeUi-uiTooling = { module = "androidx.compose.ui:ui-tooling" }
```

Comment:
should this also be anchored to the compose version?

Reply:
ya, I think we are using BOM so we shouldn't need the version refs. Let me double check